magpie/ui/home/static/style.css

```diff
@@ -54,41 +87,41 @@ input[type="submit"].button.delete {
     cursor:pointer;
 }
 
-input[type="button"].img_button.delete:hover,
-input[type="button"].img_button.delete:focus,
-input[type="submit"].img_button.delete:hover,
-input[type="submit"].img_button.delete:focus,
+input[type="button"].img-button.delete:hover,
+input[type="button"].img-button.delete:focus,
+input[type="submit"].img-button.delete:hover,
+input[type="submit"].img-button.delete:focus,
 input[type="button"].button.delete:hover,
 input[type="button"].button.delete:focus,
 input[type="submit"].button.delete:hover,
 input[type="submit"].button.delete:focus {
     background-color: #EE6363;
 }
 
-input[type="button"].img_button.warning,
-input[type="submit"].img_button.warning,
-input[type="button"].button.warning,
-input[type="submit"].button.warning {
+input[type="button"].img-button-warning,
+input[type="submit"].img-button-warning,
+input[type="button"].button-warning,
+input[type="submit"].button-warning {
     border: 1px solid #8B3A3A;
     background-color: #CD6600;
     color: white;
     vertical-align: center;
     cursor:pointer;
 }
 
-input[type="button"].img_button.warning:hover,
-input[type="button"].img_button.warning:focus,
-input[type="submit"].img_button.warning:hover,
-input[type="submit"].img_button.warning:focus,
+input[type="button"].img-button.warning:hover,
+input[type="button"].img-button.warning:focus,
+input[type="submit"].img-button.warning:hover,
+input[type="submit"].img-button.warning:focus,
 input[type="button"].button.warning:hover,
 input[type="button"].button.warning:focus,
 input[type="submit"].button.warning:hover,
 input[type="submit"].button.warning:focus {
     background-color: #FFAA33;
 }
 
-input[type="button"].img_button.cancel,
-input[type="submit"].img_button.cancel,
+input[type="button"].img-button.cancel,
```

Comment:
Not so sure where exactly in the css the cancel button is affected but before it was green and now it looks disabled (should be blue according to the theme (or red)) and not even highlighted by the mouse.
![image](https://user-images.githubusercontent.com/4202336/92623978-fa801100-f294-11ea-87cc-3477c1397507.png)
Also it turns out that both admin and anonymous users seem to be editable which I guess should be prevented (maybe not related to this PR though)

Reply:
Regarding the edit, I will disable the buttons. The API does block the operation as expected, only not reflected on UI.

As for the 'cancel', the rendered colour is the "intended" one, but I agree it can lead to confusion with the 'disabled' one. I would like to avoid the red which is associated with destructive 'delete' operations. 
To better distinguish, I think the colour of 'disabled' should instead be modified for a faded/transparent-y version of its 'enabled' state. Something to imitate following scheme:

![image](https://user-images.githubusercontent.com/19194484/92813129-9b062c00-f38e-11ea-8bde-6f13388a3b1b.png)

Reply:
with b038ad8, this is the new rendering
also found that child resources where not very clear as not available when not allowed for given service/resource types

![image](https://user-images.githubusercontent.com/19194484/92827772-3e137180-f3a0-11ea-913e-adeada70c740.png)

Reply:
After reviewing some UX sites, I agree that a gray cancel button is the way to go :P 
With your screenshot I just cannot see if moving the mouse over the button change its appearance (without highlighting, it can really appears disabled). Also, I would increase a little bit the constrast as shown on this site : https://uxmovement.com/buttons/why-cancel-buttons-should-never-have-a-color/ 
With that the cancel button would be perfect as your new disabled ones!

Reply:
The combinations are as follow. Dark/light theme are controlled by Dark Reader mod which inverts colors or darkens all components. The contrasts are very subtle on the image (could be increased), but even that small change of color is very visible when passing from normal/hover.

![image](https://user-images.githubusercontent.com/19194484/92950458-ad35a800-f42a-11ea-9753-afa350ec724f.png)

Reply:
Ok good, if the hovering is working with the new rendering (not the case with the actual demo instance) that's fine with me, the contrast don't bother me that much :P

Reply:
> (not the case with the actual demo instance)

Really? I see the hover action on my side on that instance.